### PR TITLE
User friendliness

### PR DESCRIPTION
**Pull request by SmalkiGroove**
- fix codemirror sizes
- message in group services to inform about transforming services
- button that links to docktor user manual in the navbar
- delete button in group/daemon edit
- redirect into the group/daemon after creating a new one
- compute containers create command and hide it for non-admins
- UI improvements (market and more)

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "docktor",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "private": true,
-  "proxy": "http://localhost:8080/",
+  "proxy": "http://docktor:8080/",
```
Comment (thecampagnards) [inline client/package.json]: fixed

Comment (SmalkiGroove) [reply client/package.json]: :'(

File: client/src/components/Group/views/GroupForm.tsx [hunk 2]
```diff
@@ -80,11 +87,21 @@ class Group extends React.Component<IGroupProps, IGroupStates> {
             disabled={!isAdmin}
           />
           <Form.Group widths="equal">
+            <Form.Input
```
Comment (thecampagnards) [inline client/src/components/Group/views/GroupForm.tsx]: can be remove if the dsi fix it ?

File: client/src/components/Group/views/GroupServices.tsx [hunk 3]
```diff
@@ -99,6 +122,18 @@ class GroupServices extends React.Component<IGroupProps, IGroupStates> {
       </>
     );
   }
+
+  private checkLegacy = () => {
+    fetchContainers(this.props.group._id)
+      .then((containers: IContainer[]) => {
+        containers.forEach(c => {
+          if (Object.entries(c.Labels).filter(l => l[0] === "SERVICE_NAME").length === 0) {
```
Comment (thecampagnards) [inline client/src/components/Group/views/GroupServices.tsx]: if it's a legacy docktor service behind ssi does it work ?

File: client/src/components/layout/ContainersView/ContainerCard.tsx [hunk 4]
```diff
@@ -404,4 +406,20 @@ export default class ContainerCard extends React.Component<
     return false;
   };
   private allowShell = this.computeAllowShell();
+
+  private computeCreateCommand = () => {
```
Comment (thecampagnards) [inline client/src/components/layout/ContainersView/ContainerCard.tsx]: mouai je sais pas si c'est vraiment exacte

File: client/src/components/Group/views/GroupServices.tsx (hunk 3 above)
Comment (SmalkiGroove) [reply client/src/components/Group/views/GroupServices.tsx]: yes SERVICE_NAME is a docktor v2 label, SSO labels are PROJECT_NAME and SERVICE_TYPE

File: client/src/components/layout/ContainersView/ContainerCard.tsx (hunk 4 above)
Comment (SmalkiGroove) [reply client/src/components/layout/ContainersView/ContainerCard.tsx]: pourquoi ? :(

File: client/src/components/Group/views/GroupForm.tsx (hunk 2 above)
Comment (SmalkiGroove) [reply client/src/components/Group/views/GroupForm.tsx]: Will see later

File: client/package.json (hunk 1 above)
Comment (SmalkiGroove) [reply client/package.json]:
```suggestion
  "proxy": "http://localhost:8080/",
```

**Pull request merged**